Views/MainWindow.axaml.cs

```diff
@@ -559,6 +560,33 @@ await Dispatcher.UIThread.InvokeAsync(() =>
     /// <param name="e">The event data associated with the Click event.</param>
     [SuppressMessage("ReSharper", "UnusedParameter.Local", Justification = "Event handler signature requires these parameters")]
     private void OnExitClick(object? sender, RoutedEventArgs e) => Close();
+    /// <summary>
+/// Opens the application's log file directory (%APPDATA%\MermaidPad) in the system's file explorer.
```

Comment:
hi @Mustansir-06 - thanks for your contribution! Could you please change this from an EventHandler to MVVM pattern - I updated the CONTRIBUTING.md file to include some notes there. You would need a similar method in `MainWindowViewModel` instead of the code-behind here. I have a link in the CONTRIBUTING.md with an example of where this is used already.